### PR TITLE
Feat/notifications

### PR DESCRIPTION
**Pull request by ninadarbaidze**

### CODE REVIEW

File: components/shared/NotificationItem/NotificationItem.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Image from 'next/image';
+import { Liked, QuoteIcon } from 'components/icons';
+import { useNotificationItem } from './useNotificationItem';
+import { format } from 'timeago.js';
+
+const NotificationItem: React.FC<any> = (props) => {
```
Comment (GigaGiorgadze) [inline components/shared/NotificationItem/NotificationItem.tsx]: no anyyyyyyyyyyy

File: components/shared/NotificationItem/NotificationItem.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Image from 'next/image';
+import { Liked, QuoteIcon } from 'components/icons';
+import { useNotificationItem } from './useNotificationItem';
+import { format } from 'timeago.js';
```
Comment (GigaGiorgadze) [inline components/shared/NotificationItem/NotificationItem.tsx:5]: u sure this needs .js extenstion?

File: services/quoteRequests.ts [hunk 3]
```diff
@@ -89,3 +89,40 @@ export const commentPost = async (
   });
   return response;
 };
+
+export const getNotifications = async (
+  token: string
+): Promise<AxiosResponse<any, string>> => {
```
Comment (GigaGiorgadze) [inline services/quoteRequests.ts]: no any =_=

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: components/shared/NotificationItem/NotificationItem.tsx (hunk 1 above)
Comment (ninadarbaidze) [reply components/shared/NotificationItem/NotificationItem.tsx]: გამეპარააააა :D

File: components/shared/NotificationItem/NotificationItem.tsx (hunk 2 above)
Comment (ninadarbaidze) [reply components/shared/NotificationItem/NotificationItem.tsx:5]: yep, that's the exact name of package

**Review by GigaGiorgadze — APPROVED** [review]

**Pull request merged**